src/test/java/core/basesyntax/validation/DataValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.validation;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DataValidatorTest {
+    private final DataValidator dataValidator = new DataValidator();
+
+    @Test
+    public void checkForEmptyList_validate_Ok() {
+        List<String> inputData = Arrays.asList("type,fruit,quantity");
```

Comment:
The test is supposed to check the validator's behavior when presented with an empty list, but the provided list is not empty. An empty list should be used for this test.